src/test/resources/com.demoqa.shop/features/AccountUpdate.feature

```diff
@@ -0,0 +1,59 @@
+@Run
+@Before
```

Comment:
What for @Before annotation was added?

Reply:
In plain text open home page and remove test site message. 
 @Before
    public void before(Scenario scenario) {
        log.info("Execution of scenario '{}' has been started", scenario.getName());
        ScreenshotUtil.setScenario(scenario);
        WebDriver browser = Browser.getBrowser();
        HomePage homePage = new HomePage(browser);
        homePage.clickDismiss();
        ScenarioContext.getInstance().setContext(Context.CURRENT_PAGE, homePage);
        log.info("'Before' block has been executed");

Reply:
if Before is a custom made annotation, I would suggest changing it's name as Before in JUnit has another meaning, it can be confusing. For example @ InstantiateHomePage